docs/releases/ez_platform_v3.0.md

```diff
@@ -0,0 +1,281 @@
+# eZ Platform v3.0
+
+**Version number**: v3.0
+
+**Release date**: TBD
+
+**Release type**: Fast Track
+
+## Overview
+
+## Notable changes
+
+### Symfony 5
+
+The version 3.0 moves eZ Platform to Symfony 5.0 from the previously used Symfony 3.
+
+This entails several changes to the way projects are organized.
+For details, see [Symfony 4.0](https://github.com/symfony/symfony/blob/master/UPGRADE-4.0.md)
+and [Symfony 5.0 upgrade documentation](https://github.com/symfony/symfony/blob/master/UPGRADE-5.0.md)
+
+### Field Type creation
+
+You can now use [Generic Field Type](../guide/extending_field_type.md) as a template for your custom Field Types.
+
+### Using Events instead of SignalSlots
+
+The application now uses Symfony Events instead of SignalSlots.
+The application triggers two Events per operation: one before and one after the relevant thing happens
+(see for example [BookmarkEvents](https://github.com/ezsystems/ezpublish-kernel/blob/master/eZ/Publish/Core/Event/Bookmark/BookmarkEvents.php)).
+
+To use Symfony Events, create [Event Listeners](https://symfony.com/doc/5.0/event_dispatcher.html) in your code.
+
+### New bundles
+
+The list of bundles in v3.0 has been extended by the following ones:
+
+- `ezplatform-calendar`
+- `ezplatform-content-comparison`
+- `ezplatform-workflow`
```

Comment:
I think we need to also add `ezplatform-site-factory`
https://github.com/ezsystems/ezplatform-ee/blob/master/composer.json#L41

Reply:
We will, as well as ezplatform-kernel